conf/application.conf

```diff
@@ -122,6 +122,8 @@ PROD {
   publish_events.queue_url="https://sqs.eu-west-1.amazonaws.com/163592447864/publish-events-PROD"
 }
 
+faciatool.sns.tool_topic_arn="arn:aws:sns:eu-west-1:163592447864:facia-CODE-FrontsUpdateSNSTopic-RepwK3g95V3w"
```

Comment:
For some reason I'm a bit surprised this (along with the other values) is hardcoded here but I guess it's ok!

Reply:
Yeah, let me know if you think it should live somewhere else, but this seemed like the right place for it.

Reply:
Looks like it to me, as it seems we're happy for other urls to be there!